### PR TITLE
Add motasimmakki

### LINKED ISSUE #631: Motasim
Screenshot (600)

### PR DESCRIPTION
**Pull request by motasimmakki**
resolve #631

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1217,4 +1217,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/54446880/127509715-b594b68d-4703-4bc2-aed5-1172d7df6a5c.png",
     "issueNumber": 627
   }
```
Comment (deephunt3r) [inline data.json:1219]: },

**Review by deephunt3r — CHANGES_REQUESTED** [review]
Pls update with requested changes

**Review by CodingSpecies — APPROVED** [review]
LGTM, I think this can be merged now @schmelto

**Comment by motasimmakki** [comment]
> LGTM, I think this can be merged now @schmelto

Sounds good ✌🏼

**Review by deephunt3r — APPROVED** [review]
LGTM 👍🏻 Thanks for the contribution 🎊

**Comment by CodingSpecies** [comment]
Just realised that there is conflicts that need resolving

**Review by CodingSpecies — APPROVED** [review]
LGTM.

**Comment by motasimmakki** [comment]
> LGTM.

great ✨

File: data.json [hunk 2]
```diff
@@ -1218,6 +1218,11 @@
     "issueNumber": 627
   },
   {
+    "name": "Motasim",
+    "githubUsername": "motasimmakki",
+    "imageUrl": "https://user-images.githubusercontent.com/44056349/127783862-dfa186c1-478a-45d0-8088-59cab12c36a9.png",
+    "issueNumber": 631
+  },
```
Comment (schmelto) [inline data.json:1225]:
```suggestion
  },
  {
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
lgtm

**Comment by schmelto** [comment]
Thanks for your contribution 🚀
@CodingSpecies @deephunt3r  thanks for reviewing

**Pull request merged**